resources/language/resource.language.pt_pt/strings.po

```diff
@@ -301,15 +301,15 @@ msgstr "Reproduzir"
 
 msgctxt "#30140"
 msgid "Adding contents to library"
-msgstr "A adcionar conteúdos na biblioteca"
+msgstr "Adicionando conteúdos à biblioteca"
 
 msgctxt "#30141"
 msgid "Updating library"
-msgstr "A atualizar biblioteca"
+msgstr "Atualizando a biblioteca"
 
 msgctxt "#30142"
 msgid "Re-building library"
-msgstr "A reconstruir biblioteca"
+msgstr "Reconstruindo a biblioteca"
```

Comment:
Please revert as the previous phrase is more common/used in PT-PT (we usually don't use gerund, specially when starting sentences)